src/components/ShareList.tsx

```diff
@@ -1,40 +0,0 @@
-import { h } from "preact";
```

Comment:
Are you implementing Share a post? from this, it looks like you deleted the file where the implementation was

Reply:
I changed the way I was making shareable links to use the PostPage.tsx file that you had made, so I deleted this ShareList since I was no longer using it.